src/test/resources/net/sf/jabref/logic/importer/fileformat/MsBibImporterTest3.bib

```diff
@@ -1,10 +1,12 @@
+% Encoding: UTF-8
+
 @book{raey,
   author = {5},
-  journal = {Wirtschaftsinformatik},
+  journaltitle = {Wirtschaftsinformatik},
```

Comment:
I don't see biblatex mode activated for that file (respective `jabref-comment` missing). How can this test pass? Because of the autodetection of the bib file mode?